web/src/components/ExpandedViewMode/EVMiddleColumn/TabContent/EvChildren/EvChildren.tsx

```diff
@@ -19,8 +19,10 @@ const EvChildren: React.FC<EvChildrenProps> = ({
       <EvReadOnlyHeading
         headingText={outcomeContent}
         // @ts-ignore
-        overviewIcon={<Icon name="hierarchy.svg" />}
-        overviewText={`${directChildren.length} children`}
+        overviewIcon={<Icon name="hierarchy.svg" className="not-hoverable" />}
+        overviewText={`${directChildren.length} child${
+          directChildren.length > 1 ? 'ren' : ''
```

Comment:
```suggestion
          directChildren.length !== 1 ? 'ren' : ''
```
good thinking. one improvement on this is to `!== 1` instead because that catches the `0` case, which shares grammar with the > 1 case.

Reply:
if the outcome has 0 children the "children" tab should not be displayed on the left menu.